cfme/configure/access_control.py

```diff
@@ -1386,7 +1317,8 @@ def update(self, updates):
             as 'Save' button will not be active
         """
         view = navigate_to(self, 'Edit')
-        changed = view.fill(updates)
+        wait_for(lambda: view.is_displayed, num_sec=5)
```

Comment:
@mfalesni this shouldn't be necessary should it?

Reply:
It was failing with the error 
` 'MoveTargetOutOfBoundsException: Message: Despite all the workarounds, scrolling to Input(locator=u'.//*[(self::input or self::textarea) and @name="description"]') was unsuccessful'
`
it was filling the form quick, so added a wait_for